src/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -95,11 +95,35 @@
  * @author mccowan@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = IlluminaBasecallsToSam.USAGE,
-        usageShort = IlluminaBasecallsToSam.USAGE,
+        usage = IlluminaBasecallsToSam.USAGE_SUMMARY + IlluminaBasecallsToSam.USAGE_DETAILS,
+        usageShort = IlluminaBasecallsToSam.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class IlluminaBasecallsToSam extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Transforms a lane of Illumina data file formats (*.bcl, *.locs, *.clocs, *.qseqs, etc.) into the" +
```

Comment:
"a lane of Illumina data file formats" is an odd phrasing; please rephrase. Should be more straightforward eg "Generates a SAM or BAM file from Illumina basecalling data per-lane"